### PR TITLE
Task: extract saml from url

### PR DESCRIPTION
**Pull request by Jake-Lindvall**
Two main things here:
* Non-breaking `TODO`-level changes that let you play around with `SAMLResponse` in the URL on any page load
* Fix to `useQuery()` that addresses a silent and derpy mistake I made earlier

This is not a complete [ticket](https://edma.atlassian.net/secure/RapidBoard.jspa?rapidView=1&projectKey=KIR&modal=detail&selectedIssue=KIR-1) but has some things that Abby, Selcuk, etc. may benefit from being in `dev`

### CODE REVIEW

File: src/Hooks/customHooks.js [hunk 1]
```diff
@@ -3,9 +3,8 @@ import { useLocation } from 'react-router-dom';
 // Custom hook for query string parsing from URL
 // Useful for things like Search, which normally can't depend
 // only on a text input (i.e. someone might want a shareable URL)
-export const useQuery = (urlPath, param) => {
-  // `urlPath` is the path string that comes after "{BASE_URL}/"
-  // `param` is the key of the query parameter whose value we want
-  const path = useLocation()[urlPath];
+// `param` is the key of the query parameter whose value we want
+export const useQuery = param => {
```
Comment (Jake-Lindvall) [inline src/Hooks/customHooks.js:7]: To anyone who's wondering - `useLocation()` always has a field called `search`. I thought that this was specific to the route you were on, so I thought you needed to pass in a parameter for that (it just so happened my previous work was on `/search`). 

Anyway, I was very wrong! Please forgive the stupidity haha

Comment (ghost) [reply src/Hooks/customHooks.js:7]: shhhh its fine you're learning

**Review by ghost — DISMISSED** [review]
LGTM, I'll need to test this out with the current work I have on my task branch

File: src/Hooks/customHooks.js (hunk 1 above)
Comment (SelcukEdma) [reply src/Hooks/customHooks.js:7]: thanks for the explanation 👍

**Review by SelcukEdma — DISMISSED** [review]

**Review by ghost — APPROVED** [review]
LGTM

**Review by sfowles — APPROVED** [review]

**Pull request merged**